### PR TITLE
Update user guide

### PR DESCRIPTION
**Pull request by zhaojj2209**
Partially addresses #131.

Updated user guide and standardised formatting.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -13,135 +13,169 @@ title: User Guide
...
+</div>
 
-#### 2.1.4 Analytics Tab
+--------------------------------------------------------------------------------------------------------------------
+## 2. Quick start
 
-Displays visualizations of your financial habits.
-This tab allows you analyze your spending and saving habits through charts.
+This section will show you the various components that make up Fine\\$\\$e's user interface.
+You can also follow our step-by-step guide on how to install Fine\\$\\$e and get it to work on your computer.
```
Comment (ianyong) [inline docs/UserGuide.md:65]: Should these statements be reordered to fit the subsections?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -13,135 +13,169 @@ title: User Guide
...
 
-1. Download the latest `finesse.jar` from [here](https://github.com/AY2021S1-CS2103T-W16-3/tp/releases).
+1. Download the latest version of `finesse.jar` from [here](https://github.com/AY2021S1-CS2103T-W16-3/tp/releases).
 
 1. Copy the file to the folder you want to use as the *home folder* for Fine$$e.
 
 1. Double-click the file to start the app.
-The GUI similar to the below should appear in a few seconds, with sample data included.
+The Fine$$e GUI should appear in a few seconds, with sample data included, similar to the below image.
    ![Ui](images/Ui.png)
```
Comment (ianyong) [inline docs/UserGuide.md:79]: Add a space before this line so that the image isn't so cramped?

**Review by yongping827 — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 3]
```diff
@@ -230,25 +250,23 @@ Expected Outcome:
...
 
-> :bulb: &nbsp; To specify an edit that removes all categories, use the parameter `c/` with no category name.
+> :bulb: &nbsp; To remove all categories from an expense, simply use `c/` with no category name following it.
 
 Examples:
-- `edit 1 a/5 d/22/09/2020` edits the first expense in the expenses list to have amount `$5.00` and date `22/09/2020`.
-The rest of the expense information remains unchanged.
-- `edit 3 a/$2000 c/` edits the third expense in the expenses list to have amount `$2000.00` and no categories.
-The rest of the expense information remains unchanged.
+* `edit 2 t/Shoppee Internship d/24/10/2020`
```
Comment (ianyong) [inline docs/UserGuide.md:268]:
```suggestion
* `edit 2 t/Shopee Internship d/24/10/2020`
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -299,95 +322,104 @@ Expected Outcome:
...
-adds a new income titled `Internship`, with amount `$560.00`, date `03/10/2020`, and a single category `Work`.
-- `addi t/Angpao money a/$20 d/25/01/2020 c/CNY c/Gift`
-adds a new income titled `Angpao money`, with amount `$20.00`, date `25/01/2020`, and two categories `CNY` and `Gift`.
+* `add-income t/Waitressing a/80 c/Work`
+* `addi t/Angpao money a/$20 d/25/01/2020 c/CNY c/Gift`
 
 Example Usage:
 ```
-add-income t/Internship a/560 d/03/10/2020 c/Work
+add-income t/Shoppee Internship a/560 d/03/10/2020 c/Work
```
Comment (ianyong) [inline docs/UserGuide.md:399]:
```suggestion
add-income t/Shopee Internship a/560 d/03/10/2020 c/Work
```

File: docs/UserGuide.md [hunk 5]
```diff
@@ -299,95 +322,104 @@ Expected Outcome:
...
 ```
 
 Expected Outcome:
 ```
-New income added: Internship Amount: $560.00 Date: 03/10/2020 Categories: [Work]
+New income added: Shoppee Internship Amount: $560.00 Date: 03/10/2020 Categories: [Internship]
```
Comment (ianyong) [inline docs/UserGuide.md:404]:
```suggestion
New income added: Shopee Internship Amount: $560.00 Date: 03/10/2020 Categories: [Internship]
```

File: docs/UserGuide.md [hunk 6]
```diff
@@ -299,95 +322,104 @@ Expected Outcome:
...
 ```
+Adds a new income titled `Shoppee Internship`, with amount `$560.00`, date `03/10/2020`, and a single category `Internship`.
```
Comment (ianyong) [inline docs/UserGuide.md:406]:
```suggestion
Adds a new income titled `Shopee Internship`, with amount `$560.00`, date `03/10/2020`, and a single category `Internship`.
```

**Review by ianyong — CHANGES_REQUESTED** [review]

**Review by ianyong — APPROVED** [review]
LGTM!

**Review by yongping827 — APPROVED** [review]
LGTM

**Pull request merged**